ui/components/src/InfoTooltip/InfoTooltip.tsx

```diff
@@ -41,6 +41,11 @@ export const InfoTooltip = ({
   enterDelay,
   enterNextDelay,
 }: InfoTooltipProps) => {
+  // Only wrap in a div if passed a non-element. This enables the tooltip to
+  // support text with a wrapper div while avoiding breaking css on element
+  // children by unnecessarily wrapping them.
```

Comment:
I noticed this while creating stories and trying to add styles to the buttons in the empty state. The extra wrapper div would mess up certain styling (e.g. flexbox stuff). I _think_ it existed to support the not-an-element use cases like passing just some plain text. Let me know if you think I'm missing an edge case here.

Reply:
interesting, do you know why we need the extra div in order for text element to work?